docs/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <script async type="module" src="../dist/byu-theme-components.min.js"></script>
+    <link rel="stylesheet" href="https://cdn.byu.edu/theme-fonts/1.x.x/ringside/fonts.css" media="all">
+    <link rel="stylesheet" href="https://cdn.byu.edu/theme-fonts/1.x.x/public-sans/fonts.css" media="all">
+    <link rel="stylesheet" href="lib/prism/prism.css" media="all">
+    <script type="text/javascript" src="lib/prism/prism.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        html, body {
+            height: 100%;
+            margin: 0;
+            padding: 0;
+            font-family: 'Public Sans', Arial, sans-serif;
+        }
+        h1, h2, h3, h4, h5, h6 {
+            font-family: 'HCo Ringside Narrow SSm', Arial, sans-serif;
+        }
+        .containing-element {
+            display: flex;
+            flex-direction: column;
+            height: 100%;
+        }
+
+        .page-content {
+            flex-grow: 1;
+            max-width: 1200px;
+            margin: 0 auto;
+        }
+    </style>
+    <title>Components - v2</title>
+</head>
+<body>
+<div class="containing-element">
+    <byu-header>
+        <span slot="site-title">Components v2</span>
+        <byu-menu slot="nav">
+            <a href="index.html" class="active">Home</a>
+            <a href="byu-header.html">Header</a>
+            <a href="byu-menu.html">Menu</a>
+            <a href="byu-search.html">Search</a>
+            <a href="byu-user-info.html">User Login</a>
+            <a href="byu-footer.html">Footer</a>
+        </byu-menu>
+    </byu-header>
+    <div class="page-content">
+        <h1>Welcome to v2</h1>
+        <h2>Getting Started</h2>
+        <p>
+            Add the following to your <code class="language-html">&lt;head&gt;</code>:
+        </p>
+        <pre><code class="language-html">
+            &lt;script async type="module" src="https://cdn.byu.edu/byu-theme-components/1.x.x/byu-theme-components.min.js"&gt;&lt;/script&gt;
+            &lt;link rel="stylesheet" href="https://cdn.byu.edu/theme-fonts/1.x.x/ringside/fonts.css" media="all"&gt;
+            &lt;link rel="stylesheet" href="https://cdn.byu.edu/theme-fonts/1.x.x/public-sans/fonts.css" media="all"&gt;
```

Comment:
Do any of the components use Public Sans? If not we should mention that it's a recommended font, but not required for the components to function.

Reply:
Yes. The footer and menu.